collector/src/compile/benchmark/profile.rs

```diff
@@ -12,13 +12,15 @@ pub enum Profile {
     Debug,
     Doc,
     Opt,
+    Clippy,
 }
 
 impl Profile {
     pub fn all() -> Vec<Self> {
```

Comment:
This will need to be modified in `BenchNext` and `bench_published_artifact`. These two functions should not benchmark Clippy. Probably the functions should directly decide the specific profiles to benchmark, rather than delegating to these two functions (`all` and `all_non_doc`).